vscode_projects/__init__.py

```diff
@@ -0,0 +1,78 @@
+"""
+This extension provides a quick introduction on how to use the new Python pluging interface.
+Hope you like it.
+"""
+
+from albert import *
+import os
+
+import json
+import urllib.parse
+
+md_iid = "0.5"
+md_version = "1.1"
+md_name = "vscode projects"
+md_description = "vscode projects"
```

Comment:
This is a human readable pretty printed string. i should be brief and imperative like "Open VSCode projects"